.travis.yml

```diff
@@ -31,7 +32,7 @@ matrix:
           env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest --ignore-platform-reqs" COVERAGE=true TEST_COMMAND="composer test-ci"
 
 before_install:
-    - travis_retry composer self-update
+    - if [[ ! $COVERAGE = 'true' && ! $TRAVIS_PHP_VERSION = hhvm* ]]; then phpenv config-rm xdebug.ini || echo "xdebug not available"; fi
```

Comment:
Please use this as a boilerplate: https://github.com/php-http/boilerplate/blob/master/.travis.yml#L31-L36